src/helpers/errors.ts

```diff
@@ -1,38 +1,50 @@
 import { HTTPError, NetworkError } from '@opensrp/server-service';
-import { toast } from 'react-toastify';
+import { toast, ToastOptions } from 'react-toastify';
 import { ACCESS_DENIED, NETWORK_ERROR, USER_HAS_NO_VALID_ASSIGNMENTS } from '../configs/lang';
 import { growl } from './utils';
 
 /** union of all ErrorTypes that we are working with */
 type ServiceError = HTTPError | Error | NetworkError;
 
+/** autoClose types */
+type AutoClose = 5000 | false;
+
 /**
  * Display error message using growl
  * @param error - the error object
  * @param customMessage - custom error message
+ * @param autoClose - Delay in ms to close the toast. If set to false,
+ * the notification needs to be closed manually
  */
-export const displayError = (error: ServiceError, customMessage: string = '') => {
+
+export const displayError = (
+  error: ServiceError,
+  customMessage: string = '',
+  autoClose: AutoClose = 5000
+) => {
   /** opensrp error object for plans after filtering plans by a user when the assignments are expired
```

Comment:
lets also remove this string. looks like it describes something that was removed